src/containers/ToolboxSubject/ToolboxSubjectPage.tsx

```diff
@@ -0,0 +1,142 @@
+  const { subjectId, topicList } = getUrnIdsFromProps({
+    ndlaFilm: false,
+    match,
+  });
+
+  const refs = topicList.map(() => React.createRef<HTMLDivElement>());
+  const [selectedTopics, setSelectedTopics] = useState<GQLTopic[]>([]);
+  const { loading, data } = useGraphQuery<Data>(subjectPageQuery, {
+    variables: {
+      subjectId,
+    },
+  });
+
+  useEffect(() => {
+    topicList.forEach((topicId: string) => {
+      const alreadySelected = selectedTopics.find(
+        (topic: GQLTopic) => topic.id === topicId,
+      );
+      if (!alreadySelected) {
+        const exist = subject.allTopics.find(
+          (topic: GQLTopic) => topic.id === topicId,
+        );
+        if (exist) setSelectedTopics([exist, ...selectedTopics]);
+      }
+    });
+    scrollToTopic(topicList.length - 1);
+  });
+
+  const scrollToTopic = (index: number) => {
+    const ref = refs[index];
+    if (ref && ref.current) {
+      const positionFromTop =
+        ref.current.getBoundingClientRect().top +
+          document?.documentElement?.scrollTop || 100;
+      window.scrollTo({
+        top: positionFromTop - 100,
+        behavior: 'smooth',
+      });
+    }
+  };
+
+  if (loading) {
+    return null;
+  }
+
+  if (!data) {
+    return <DefaultErrorMessage />;
+  }
+
+  const subject = data.subject;
+
+  const topics = subject.topics?.map((topic: GQLTopic) => {
+    return {
+      ...topic,
+      label: topic.name,
+      selected: topic.id === topicList[0],
+      url: toTopic(subject.id, topic.id),
+    };
+  });
+
+  const onSelectTopic = (index: number, id?: string) => {
+    const exist = subject.allTopics?.find((topic: GQLTopic) => topic.id === id);
+    if (exist) {
+      if (index === 0) {
+        setSelectedTopics([exist]);
+      } else if (index > 0) {
+        setSelectedTopics([...selectedTopics.slice(0, index), exist]);
+      }
+    }
+  };
+
+  const TopicBoxes = () => (
+    <>
+      {selectedTopics.map((topic: GQLTopic, index: number) => {
+        return (
+          <div key={index} ref={refs[index]}>
+            <ToolboxTopicWrapper
+              subjectId={subject.id}
+              topicId={topic.id}
+              locale={locale}
+              onSelectTopic={onSelectTopic}
+              topicList={topicList}
+              index={index}
+            />
+          </div>
+        );
+      })}
+    </>
+  );
+
+  if (!topics) {
+    return;
+  }
+
+  return (
+    <OneColumn className={''}>
+      <ToolboxInfo
+        topics={topics}
+        onSelectTopic={(_e: React.MouseEvent<HTMLElement>, id?: string) =>
+          onSelectTopic(0, id)
+        }
+        title="Verktøykassa"
+        introduction="Hva vil det si å arbeide utforskende? Hvordan kan du lære bedre? Hva skal til for å få gruppearbeid til å fungere? I Verktøykassa finner både elever og lærere ressurser som er aktuelle for alle fag, og som støtter opp under læringsarbeid og utvikling av kunnskap, ferdigheter og forståelse."
```

Comment:
i18n. Her må det kunne vises subject.name i stedet for kun Verktøykassa. Då vil også cypress-testen funke som den var.